README.md

```diff
@@ -210,7 +210,14 @@ enough for most real-life use cases such as receipts. Some news sites may need
 even 2GB of RAM.*
 
 
-#### 2. Local development
+#### 2. Run in Docker
+
+* `docker build -t example/url-to-pdf .`
+* `docker run -it --env-file ./.env.sample -p 9000:9000 example/url-to-pdf`
```

Comment:
This references to `.env.sample` file instead of the `docker.env.sample`. Also see the comment about docker.env.sample file.

About the port binding: could they be taken from `$PORT` env var? It should be the master configuration to set which port server listens to even you'd run it in Docker.